src/lib/SX1280Driver/SX1280.cpp

```diff
@@ -465,9 +465,10 @@ void ICACHE_RAM_ATTR SX1280Driver::TXnbISR()
     TXdoneCallback();
 }
 
-void ICACHE_RAM_ATTR SX1280Driver::TXnb(uint8_t * data, uint8_t size, SX12XX_Radio_Number_t radioNumber)
+void ICACHE_RAM_ATTR SX1280Driver::TXnb(uint8_t * data, uint8_t size, SX12XX_Radio_Number_t radioNumber, SX12XX_Radio_Number_t clearChannelsMask)
 {
-    if (currOpmode == SX1280_MODE_TX) //catch TX timeout
+    //catch TX timeout and no transmission in case of LBT channel in use
+    if ((currOpmode == SX1280_MODE_TX) || (!clearChannelsMask))
```

Comment:
Sorry I havent tested this yet, but Im just going through the logic in my head.

`transmittingRadio & clearChannelsMask` needs to be kept in rx_main.cpp so that the correct radios are set to SX1280_MODE_TX.  Remember there are 2 radios in a Gemini Tx and if 1 isnt clear the other can still send.

So you can remove the need for adding `clearChannelsMask` to `TXnb` and change this line to the below.

`if ((currOpmode == SX1280_MODE_TX) || (!radioNumber))`

If no radios are set to transmit (radioNumber==0) then execute TXnbISR etc.